MANIFEST.in

```diff
@@ -0,0 +1,2 @@
+recursive-include server/app/web/templates *
```

Comment:
My knowledge of this area is pretty sketchy but my understanding is that MANIFEST.in includes these files into your package, but does not actually write them out to the installed space.  However, you have include_package_data=True, which _might_ do what you need.

Can you verify that these are actually getting installed?

Reply:
Sure, I'll double check that.

Reply:
Yes, these are being installed.